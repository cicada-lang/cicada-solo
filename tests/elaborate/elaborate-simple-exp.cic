@elaborate 123
@elaborate "abc"
@elaborate add1(123)
