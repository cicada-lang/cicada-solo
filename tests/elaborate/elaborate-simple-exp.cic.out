123: Nat
"abc": String
124: Nat
